src/ontology/oba.Makefile

```diff
@@ -139,14 +160,24 @@ $(REPORTDIR)/%.tsv: ../sparql/synonyms-exact.sparql $(TMPDIR)/%.owl
 prepare_oba_alignment: $(REPORTDIR)/uberon.tsv $(REPORTDIR)/efo.tsv $(REPORTDIR)/pato.tsv $(REPORTDIR)/oba.tsv $(REPORTDIR)/vt.tsv $(REPORTDIR)/cl.tsv $(REPORTDIR)/go.tsv $(REPORTDIR)/chebi.tsv
 	echo "OK cool all tables prepared."
 
-CHECK_SPARQL=oba.owl
+#This is the OBA-VT DIFF Pipeline
+
+$(TMPDIR)/%.owl:
+	wget $(OBOBASE)/$*.owl -O $@
+
+$(TMPDIR)/simple-%.owl: $(TMPDIR)/%.owl data/vtoba_terms.txt 
+	$(ROBOT) merge -i $< \
+	filter -T data/vtoba_terms.txt  --select 'self annotations' --signature true -o $@
+
+$(TMPDIR)/merge-oba-vt.owl: $(TMPDIR)/oba.owl $(TMPDIR)/vt.owl
+	$(ROBOT) merge -i $< -i $(TMPDIR)/vt.owl -o $@
+
+
+$(REPORTDIR)/oba-vt-diff-simple.yaml: $(TMPDIR)/simple-oba.owl $(TMPDIR)/simple-vt.owl
```

Comment:
Hi, @cmungall. I'm trying to generate diff via mappings using oba and vt ontologies.
In order to get more filtered results, I filtered both ontologies using robot and passed my desired terms as input (they are in data/vtoba_terms.txt).
The generated file was then used as input for diff-via-mappings oak.
The result of this process is a diff file with only a few lines, as shown below:
```
category: Identical
left_object_id: OBA:VT0100015
left_object_label: circulatory system development trait
left_predicate_id: rdfs:subClassOf
left_subject_id: OBA:VT0000121
left_subject_label: cardiovascular system development trait
right_object_id: VT:0100015
right_object_label: circulatory system development trait
right_predicate_ids:
- rdfs:subClassOf
right_subject_id: VT:0000121
right_subject_label: cardiovascular system development trait

```
I was expecting the diff result will generate more entries.
Am I doing something wrong?
Thank you very much.

Reply:
simple-oba.owl is almost devoid of logical axioms, it is a big flat list with a single subClassOf axiom.

Currently the diff algorithm is intentionally asymmetrical, it will compare every edge in the left ontology (simple-oba.owl), here there is only one edge. The use case here is edge transferrence, from one ontology to another.

If you can fix the upstream robot to include all of the edges (I assume you just want is-a edges), I can also add an option for matching in both directions

Reply:
https://github.com/INCATools/ontology-access-kit/issues/404